server/test/suite_localization.cpp

```diff
@@ -17,10 +19,113 @@ class LocalizationFixture : public ::testing::Test
 
 protected:
 	Localization localization_;
+    std::shared_ptr<DataStore> data_store_ = std::make_shared<DataStore>();
 };
 
-TEST_F(LocalizationFixture, Init)
+/*
+ * TEST: Localize
+ * EXPECT: <x,y,z> of dummy node.
+*/
+TEST_F(LocalizationFixture, RoundTripLocalizationTest)
```

Comment:
Can you specify what you are testing and how? You can use a naming convention such as `SoftwareUnderTest_WhenSomethingHappens_WillPerformAction`.

Reply:
yes, sure.